ui/raidboss/data/05-shb/trial/memoria_misery_ex.js

```diff
@@ -0,0 +1,139 @@
+      regexDe: Regexes.startsUsing({ id: '4CC6', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4CC6', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4CC6', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
+      delaySeconds: 6,
+      response: Responses.getIn(),
+    },
+    {
+      id: 'Varis Loaded Gunshield (Spread)',
+      regex: Regexes.ability({ id: '4CD8', source: 'Varis Yae Galvus', capture: false }),
+      regexDe: Regexes.ability({ id: '4CD8', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.ability({ id: '4CD8', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.ability({ id: '4CD8', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
+      delaySeconds: 6,
+      response: Responses.spread(),
+    },
+    {
+      id: 'Varis Reinforced Gunshield (Block)',
+      regex: Regexes.startsUsing({ id: '4CD9', source: 'Varis Yae Galvus', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4CD9', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4CD9', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4CD9', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
+      delaySeconds: 18,
+      infoText: {
+        en: 'Watch out Blocking',
+        ja: 'ブロックしない側に攻撃',
+        cn: '攻击未格挡的方向',
+      },
+    },
+    {
+      id: 'Varis Electrified Gunshield (Knockback)',
+      regex: Regexes.ability({ id: '4CD7', source: 'Varis Yae Galvus', capture: false }),
+      regexDe: Regexes.ability({ id: '4CD7', source: 'Varis yae Galvus', capture: false }),
+      regexFr: Regexes.ability({ id: '4CD7', source: 'Varis yae Galvus', capture: false }),
+      regexJa: Regexes.ability({ id: '4CD7', source: 'ヴァリス・イェー・ガルヴァス', capture: false }),
+      delaySeconds: 20,
+      response: Responses.knockback(),
+    },
+    {
+      id: 'Varis Magitek Spark (Spread)',
+      regex: Regexes.startsUsing({ id: '4E50', source: 'Gunshield', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4E50', source: 'Gewehrschild', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4E50', source: 'bouclier-canon', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4E50', source: 'ガンシールド', capture: false }),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Varis Magitek Torch (Stack)',
+      regex: Regexes.startsUsing({ id: '4E4F', source: 'Gunshield', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4E4F', source: 'Gewehrschild', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4E4F', source: 'bouclier-canon', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4E4F', source: 'ガンシールド', capture: false }),
+      response: Responses.stack(),
+    },
+  ],
+  timelineReplace: [
+    {
+      'locale': 'ja',
```

Comment:
These text are all extract from client, I included Japanese only because only English and Japanese that I can recognize.

Except the `what-Est Action` (which I translate it as `XXX発動`), npm test tells me that it is not allowed when one item of `replaceText` can be prefix of the other. But I prefer it can annouce the Est action timing.

Reply:
Just making sure. The general preference is for any translations to match what the user would see in-game. As for the `-Est` stuff, I think we can better help you once you submit the timeline in its own pull request.